content/administration/upgrade.rst

```diff
@@ -1,264 +1,340 @@
-via the `Odoo Support page <https://www.odoo.com/help>`_.
+         $ python <(curl -s https://upgrade.odoo.com/upgrade) test -d <your db name> -t <target version>
 
-Under the *Ticket Description* section, select *An issue related to my upgrade* ticket type.
+      The following command can be used to display the general help and the main commands:
 
-   .. image:: upgrade/test-assistance.png
-      :align: center
-      :alt: Selection of "An issue related to my upgrade" as Ticket Type in the support form on Odoo
+      .. code-block:: console
 
-   .. warning::
-      If you choose another *Ticket Description* type, the request will be redirected to another
-      team. This will slow down the processing and response time.
+         $ python <(curl -s https://upgrade.odoo.com/upgrade) --help
 
-Please provide as much detail as you can (i.e., videos and screenshots to illustrate your issue).
-This will avoid clarifying questions and speed up the resolution process significantly.
+      An upgraded test database can also be requested via the `Upgrade page
+      <https://upgrade.odoo.com/>`_.
+
+      .. note::
+         - For security reasons, only the person who submitted the upgrade request can download it.
+         - For storage reasons, the database's copy is submitted without a filestore to the upgrade
+           server. Therefore, the upgraded database does not contain the production filestore.
+         - Before restoring the upgraded database, its filestore must be merged with the production
+           filestore to be able to perform tests in the same conditions as it would be in the new
+           version.
+         - The upgraded database contains:
+
+           - A `dump.sql` file containing the upgraded database
+           - A `filestore` folder containing files extracted from in-database records into
+             attachments (if there are any) and new standard Odoo files from the targeted Odoo
+             version (e.g., new images, icons, payment provider's logos, etc.). This folder be
+             merged with the production filestore to get the full upgraded filestore.
 
 .. note::
-   * The purpose of the test phase is not to correct existing data or configurations in your
-     database.
-   * |assistance-contact|
+   You can request multiple test databases if you wish to test an upgrade more than once.
 
-.. _upgrade/steps-production:
+In all cases, the process is the same:
 
-The production launch
-=====================
+#. Exporting the database to a file
+#. Uploading the file to the upgrade server
+#. Running a series of :ref:`standard migration scripts <reference/upgrade/migration-scripts>` to
+   upgrade standard modules
+#. Downloading the upgraded database
+#. Importing the file into a database
+#. (*Optional*) Running a series of :ref:`custom migration scripts
+   <reference/upgrade/migration-scripts>` developed by third parties to upgrade custom modules
```

Comment:
It could be nice to give an overview of what we're gonna talk about, like a table of contents, how the process work or somthing like that, not sure how yet. But as it is right now, I'm not sure what's the added value of this here.

Reply:
IMO it helps to understand what is going on exactly behind the scenes when an upgrade happen, since it is different than the usual "update" of many applications (Word, web browser, Android/iOS app) where the new code is downloaded and the application restart. With Odoo, the data is transmitted to an other computer, it is changed significantly, and then uploaded back to the sender.

This can help external to have a better understanding on the migration (movement) of their (sensible) data

But you are not the only one raising doubts about this section, if you really think that having this numbered list will hurt us more than it will do good, I will remove it, but IMO it does not have a negative impact:  might be a positive impact, or a neutral one

Reply:
I'm thinking that a mix of your section and this one in the current doc: https://www.odoo.com/documentation/16.0/administration/upgrade.html#process-workflow
could be a bit more helpful
Talking about the whole upgrade process beginning to end and what happens in the middle with the upgrade platform. However, we should find a way to have it clear and not repeat ourselves (as it describes both testing and prod upgrades, and the "platform" process is the same for both), and also check where to add it so it's clear and not confusing :thinking: 

I suggest to leave this for after the OXP, to have more time to find the best place and words for it.

Could you remove this for the moment?
Of course, if you have some ideas already you can comment them or make a skectch for the future (not a priority before publishing the rest of the doc)

Reply:
I think we are barking at the wrong tree, this is useful but not on administration/upgrade -> I am moving it to the developer page for later use :D

Reply:
Sounds good to me !